common/djangoapps/student/forms.py

```diff
@@ -3,6 +3,9 @@
 """
 from importlib import import_module
 import re
+import socket
```

Comment:
@douglashall Cool. Do you know any sites that continue to use this view? If not, I can create a jira ticket to retire the view.